qiskit/dagcircuit/dagnode.py

```diff
@@ -73,7 +73,6 @@ def semantic_eq(node1, node2, bit_indices1=None, bit_indices2=None):
                 "release will require the mappings to be provided as arguments.",
                 DeprecationWarning,
             )
-
```

Comment:
Oops, didn't mean to do this. Doesn't seem worth a CI burn though - this code path will be deleted soon (deprecation is from 0.20)